near-plugins-derive/tests/upgradable.rs

```diff
@@ -599,10 +599,10 @@ async fn test_deploy_code_in_batch_transaction_pitfall() -> anyhow::Result<()> {
         .args_json(json!({ "function_call_args": FunctionCallArgs {
         function_name: "migrate_with_failure".to_string(),
         arguments: Vec::new(),
-        amount: 0,
-        gas: Gas::ONE_TERA,
+        amount: NearToken::from_yoctonear(0),
+        gas: Gas::from_tgas(1),
     } }))
-        .gas(near_workspaces::types::Gas::from_tgas(200));
+        .gas(near_workspaces::types::Gas::from_tgas(201));
```

Comment:
Is bumping it up from 200 to 201 necessary?

Reply:
No. It's not. Will be reverted in the [PR](https://github.com/mitinarseny/near-plugins/pull/2).

Reply:
Unfortunately, gas consumption went up a bit. It's likely due to a bit heavier logic in updated dependencies such as borsh/near-sdk/etc...